locales/en-US/editor.properties

```diff
@@ -218,6 +218,11 @@ PIXEL_UNIT=pixels
 PERCENTAGE_UNIT=percentage
 EM_UNIT=em font unit
 
+# extensions/default/InlinePaddingEditor
+
+ALL_SIDE=All Sides
+INDIVIDUAL_SIDES=Individual Sides
+SET_PADDING_FOR=Set Padding For
 # extensions/default/InlineColorEditor
```

Comment:
Can you please insert a blank line above this comment, so it sets off this section visibly?